test/__init__.py

```diff
@@ -0,0 +1 @@
+# Intentionally left empty
```

Comment:
why do this (i.e., include a comment) here, and not in `vsc/install/__init__.py` where people may actually wonder why the `extend_path` thing isn't there (it shouldn't be, and it should be very clear that's not a mistake)